Neos.ContentRepository.Core/Classes/Feature/WorkspaceCommandHandler.php

```diff
@@ -286,10 +287,10 @@ private function publishContentStream(
         }
 
         if (count($events) === 0) {
-            return null;
+            return;
         }
         try {
-            return $this->eventPersister->publishEvents(
+            $commandHandlingDependencies->publishEvents(
```

Comment:
so this seems to be the single use of this weird API, why don't we keep the EventPersister here?